orca-web/orca-web.gradle

```diff
@@ -90,7 +90,6 @@ dependencies {
   testImplementation("cglib:cglib-nodep")
   testImplementation("org.objenesis:objenesis")
   testImplementation("org.junit.jupiter:junit-jupiter-api")
-  testImplementation("org.hamcrest:hamcrest-core:1.3")
```

Comment:
I snuck this change in here, it is not related to the rest of the PR... but I also don't think we use hamcrest anywhere in orca